react-native-libraries.json

```diff
@@ -9593,5 +9593,18 @@
     ],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/JedrzejMajko/react-native-hardwired",
+    "npmPkg": "react-native-hardwired",
+    "nameOverride": "React Native Hardwired",
+    "examples": [
+      "https://github.com/JedrzejMajko/react-native-hardwired/tree/master/packages/react-native-hardwired/examples/SimpleTest"
+    ],
+    "images": ["https://github.com/JedrzejMajko/react-native-hardwired/blob/master/page/logohardwired.png"],
+    "ios": true,
+    "android": true,
+    "dev": false,
+    "expo": false
```

Comment:
Hello @JedrzejMajko, thank you for adding new entry to the directory! 👍 

```suggestion
    "githubUrl": "https://github.com/JedrzejMajko/react-native-hardwired",
    "nameOverride": "React Native Hardwired",
    "examples": [
      "https://github.com/JedrzejMajko/react-native-hardwired/tree/master/packages/react-native-hardwired/examples/SimpleTest"
    ],
    "ios": true,
    "android": true
```

Few small suggestions before we ship it:
* `npmPkg` can be skipped since it matches repository name
* we do not allow putting branding/marketing material in the `images` field
  <img width="643" alt="Screenshot 2023-06-10 191610" src="https://github.com/react-native-community/directory/assets/719641/6d7e1199-d780-4e9f-8b85-724dd120c860">
* falsy flags can be skipped